apollo-portal/src/main/java/com/ctrip/framework/apollo/portal/service/NamespaceService.java

```diff
@@ -58,7 +63,8 @@ public NamespaceService(
       final AppNamespaceService appNamespaceService,
       final InstanceService instanceService,
       final @Lazy NamespaceBranchService branchService,
-      final RolePermissionService rolePermissionService) {
+      final RolePermissionService rolePermissionService,
+      AdditionalUserInfoEnrichService additionalUserInfoEnrichService) {
```

Comment:
How about we keep the code style aligned with the previous ones?
```suggestion
      final AdditionalUserInfoEnrichService additionalUserInfoEnrichService) {
```